server/webapp/WEB-INF/rails.new/spec/controllers/admin/materials/pluggable_scm_controller_spec.rb

```diff
@@ -36,12 +36,12 @@
     end
 
     it "new" do
-      {:get => '/admin/pipelines/pipeline.name/materials/pluggable_scm/new/plugin-id'}.should route_to(:controller => 'admin/materials/pluggable_scm', :action => 'new', :pipeline_name => 'pipeline.name', :plugin_id => 'plugin-id')
+      {:get => '/admin/pipelines/pipeline.name/materials/pluggable_scm/new/plugin.id-1'}.should route_to(:controller => 'admin/materials/pluggable_scm', :action => 'new', :pipeline_name => 'pipeline.name', :plugin_id => 'plugin.id-1')
       send('admin_pluggable_scm_new_path', :pipeline_name => 'foo.bar', :plugin_id => 'plugin-id').should == '/admin/pipelines/foo.bar/materials/pluggable_scm/new/plugin-id'
     end
 
     it "create" do
-      {:post => '/admin/pipelines/pipeline.name/materials/pluggable_scm/plugin-id'}.should route_to(:controller => 'admin/materials/pluggable_scm', :action => 'create', :pipeline_name => 'pipeline.name', :plugin_id => 'plugin-id')
+      {:post => '/admin/pipelines/pipeline.name/materials/pluggable_scm/plugin.id-1'}.should route_to(:controller => 'admin/materials/pluggable_scm', :action => 'create', :pipeline_name => 'pipeline.name', :plugin_id => 'plugin.id-1')
```

Comment:
Line is too long. [234/200]<br>Use the new Ruby 1.9 hash syntax.<br>Space inside { missing.<br>Space inside } missing.